### ISSUE TITLE
ModuleNotFoundError: No module named 'pantab' with version 3.0.3

### ISSUE THREAD

**Issue by chris-park** [innobi/pantab#215]
**Describe the bug**
`ModuleNotFoundError: No module named 'pantab'` after installing `pantab==3.0.3`. Worked fine after rolling back to version `3.0.2`.

**To Reproduce**

```shell
python3.10 -m venv .venv
source .venv/bin/activate
pip install --upgrade pantab
python
```
```python
import pantab as pt
# ModuleNotFoundError: No module named 'pantab'
```

**Desktop (please complete the following information):**
 - OS: Amazon Linux 2

**Comment by jorwoods** [comment]
It appears the build system omitted files from the bundle. I have yanked the 3.0.3 release while we investigate.

**Comment by WillAyd** [comment]
@lithomas1 have you ever seen a case with cibuildwheel and meson where the install actually doesn't include any of the python files, yet everything still passes? Seems to be the situation we have currently

**Comment by lithomas1** [comment]
Wild guess but maybe it works because the checkout action cd's you inside the pantab package (so the import's work?)?

Looking at the meson.build file, you are not installing any of the pure Python files, so they don't get copied into the wheel I think.

(Look at https://github.com/pandas-dev/pandas/blob/451f1069bde6c6dfb1b09acbc45ddb1c945770b5/pandas/meson.build#L27-L52 for how to install files/directories)

**Comment by WillAyd** [comment]
I just published 3.0.4 . It doesn't have any new features, but fixes this issue and warns about features that will be removed in the 4.x series
